lab-adam/route/hospital-route.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
+
+const Router = require('express').Router;
+const jsonParser = require('body-parser').json();
+const debug = require('debug')('hospital:hospital-route');
+
+const createError = require('http-errors');
+const City = require('../model/city.js');
+const Hospital = require('../model/hospital.js');
+
+const hospitalRouter = module.exports = new Router();
+
+hospitalRouter.get('/api/city/:cityID/hospital/:id', function(req, res, next){
+  debug('running GET route');
+  Hospital.findById(req.params.id)
+  .then(hospital => res.json(hospital))
+  .catch(err => next(createError(404, err.message)));
+});
+
+hospitalRouter.put('/api/city/:cityID/hospital/:id', jsonParser, function(req, res, next){
```

Comment:
As above, no need to require the city ID. The array of hospitals on the city is not being altered as the ID doesn't change. Only the hospital is being updated.